src/main/java/org/capstone/job_fair/models/entities/company/CompanyEntity.java

```diff
@@ -16,53 +16,46 @@
 @EqualsAndHashCode(onlyExplicitlyIncluded = true)
 @Table(name = "company", schema = "dbo")
 public class CompanyEntity {
+    @ManyToMany
```

Comment:
Please move the many-to-many lists to them bottom of the variable declaration

Reply:
I used intelij format code, but I will change if you wish 🥉